packages/react-switch/src/components/Switch/useSwitchState.ts

```diff
@@ -141,6 +138,10 @@ export const useSwitchState = (state: SwitchState) => {
     [onKeyDownCallback, setChecked],
   );
 
+  useEvent(eventData.element, 'pointermove', onPointerMove);
```

Comment:
why are both DOM and synthetic pointer events used ? is it possible to use either native or React events ?